.github/workflows/ci-pre-commit.yml

```diff
@@ -4,8 +4,21 @@ on: [push, pull_request]
 
 jobs:
   build:
+    name: "Linux - pre-commit checks - Python 3.10"
+    timeout-minutes: 30
     runs-on: ubuntu-latest
+    env:
+      PRE_COMMIT_USE_MICROMAMBA: 1
     steps:
+    - name: Checkout branch
     - uses: actions/checkout@v3
```

Comment:
`quantcore.thek` uses `actions/checkout@v4` - I looked at the changes and there doesn’t seem to be anything too major, should I use the same here?

Reply:
I would stick to v3 and be consistent across actions in plateau, see https://github.com/search?q=repo%3Adata-engineering-collective%2Fplateau%20checkout&type=code.

At a later point, we can add dependabot to update actions.

Reply:
Ok - leaving it as is